### PR TITLE
CNV-3374 Fixing Content Errors (Originally BZ1678782) 

### PR DESCRIPTION
**Pull request by bgaydosrh**
See Jira story: https://jira.coreos.com/browse/CNV-3374

Note that there are queries requiring SME and QE review, so more content is coming (more commits).

Test Build: http://file.bos.redhat.com/bgaydos/110619/cnv/cnv_users_guide/

Tagging @alexxa for QE Review.

Request Peer Review, optional content review, from @aburdenthehand or @ousleyp

Style review requested from @ahardin-rh @adellape @sheriff-rh

Label **Peer Review Needed/Done** and **enterprise-4.3**.

Thanks,
Bob

### CODE REVIEW

**Comment by bgaydosrh** [comment]
FYI... I am reverting these changes this afternoon and will rebuild......

**Comment by bgaydosrh** [comment]
QE Check complete. Just needs Peer Review and then we can Merge and pick to 4.2 and 4.3.

File: modules/cnv-monitoring-vm-migration-cli.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ of the `VirtualMachineInstance` configuration.
 ----
 $ oc describe vmi vmi-fedora
 ----
-+
+
```
Comment (adellape) [inline modules/cnv-monitoring-vm-migration-cli.adoc]: This `+` should still be here, but it also needs a `+` before the `oc describe` codeblock.

**Review by adellape — CHANGES_REQUESTED** [review]

File: modules/cnv-monitoring-vm-migration-cli.adoc (hunk 1 above)
Comment (bgaydosrh) [reply modules/cnv-monitoring-vm-migration-cli.adoc]: Thanks @adellape. Done. Should be ready for merge and pick.

**Review by adellape — APPROVED** [review]
👍

**Pull request merged**